images/scripts/services.setup

```diff
@@ -35,6 +35,11 @@ EOF
 chmod 755 /root/run-freeipa
 /root/run-freeipa
 
+# HACK: Podman fails to relabel symlinks properly, accidentally relabelling their targets instead
+# https://github.com/opencontainers/selinux/pull/173
+# Disable relabelling for subsequent runs.
+sed -i 's/data:Z/data/' /root/run-freeipa
```

Comment:
Without properly labeled ffiles, we need to switch off SELinux entirely.

Reply:
I feel like it shouldn't be necessary.  The logic here is that we need to relabel the first time we run it because the empty directory should have the correct label.  After that, everything in that directory will have been created by the container itself and shouldn't need relabelling.

At the same time
 - this is the services image
 - this is a workaround for a known bug in the image
 - we have better things to work on

So :+1: to disabling selinux to deal with this.